build.gradle.kts

```diff
@@ -169,6 +169,11 @@ subprojects {
     kotlin {
       target("**/*.kt")
       ktlint(libs.versions.ktlint.get())
+      licenseHeaderFile(rootProject.file("gradle/license-header.txt"))
+      targetExcludeIfContentContainsRegex("" +
+        "// Copyright 2019-2023 the Contributors to the WASI Specification|" +
+        "// ktlint-disable filename|" +
```

Comment:
Otherwise, spotless would override the whole header get rid of the ktlint comment and fail on the next iteration.

Reply:
Are there many files that contain this header? You can exclude individual files, see how we do it e.g. in `filament/views/build.gradle`.